frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -16,18 +16,27 @@ import {
   modelFor,
   referenceFor,
 } from '@console/internal/module/k8s';
+import { getNamespace } from '../../selectors';
 
 type DropdownItemProps = {
   model: K8sKind;
   name: string;
+  namespace?: string;
 };
 
-const DropdownItem: React.FC<DropdownItemProps> = ({ model, name }) => (
+const DropdownItem: React.FC<DropdownItemProps> = ({ model, name, namespace }) => (
   <span className="co-resource-item">
-    <span>
+    <span className="co-resource-icon--fixed-width">
       <ResourceIcon kind={referenceForModel(model)} />
     </span>
-    <span className="co-truncate show co-nowrap small">{name}</span>
+    <span className="co-resource-item__resource-name">
+      <span>{name}</span>
+      {namespace && (
+        <span className="co-resource-item__resource-api text-muted co-truncate show co-nowrap small">
```

Comment:
We should generalize the `co-resource-item__resource-api` class name if we're using it in a different context.

Reply:
This class is not required at all removing the usage of it from here.